setup.cfg

```diff
@@ -23,8 +23,8 @@ python_requires = >=3.6.1
 install_requires =
     matplotlib
     numpy >=1.9.3
-    scipy
     loguru
+    pydra
     pybids
```

Comment:
Eventually, we need to move this to extras as well (it comes with SO MANY dependencies). Could you please open an issue to generically move as many dependencies as possible to extra dependencies? Not now, but it would be good to do so later.